posix.mak

```diff
@@ -321,9 +321,7 @@ CHANGELOG_DDOC=${DDOC} changelog/changelog.ddoc $(NODATETIME)
 CHANGELOG_PRE_DDOC=${CHANGELOG_DDOC} changelog/prerelease.ddoc
 CHANGELOG_PENDING_DDOC=${CHANGELOG_DDOC} changelog/pending.ddoc
 
-PREMADE=appendices.html articles.html fetch-issue-cnt.php howtos.html \
```

Comment:
There was no `articles.html` anymore.

Reply:
Some of the articles were moved to the wiki (by yours truly) so they could be more easily maintained, as many of them had rotted horribly.